plugins/opensource/emoticons.md

```diff
@@ -78,9 +78,37 @@ tinymce.init({
 });
 ```
 
+### `emoticons_database`
+
+{{ site.requires_5_6v }}
+
+This option provides the ability to specify which built-in emoji database to use when rendering emojis in the editor. The following built-in emoji databases are available:
+* `emojis` - This database uses Unicode characters to represent emoji in the editor content.
+* `emojiimages` - This database uses images provided by the Twitter Emoji (twemoji) project to represent emoji in the editor content.
```

Comment:
Could we add, something like.  

This option is recommended for users on legacy browsers that do not support modern emoji.

Reply:
No, we can't recommend this since it has potential legal implications.

Reply:
we should refrain from providing legal advice on CC-BY 4.0, but we can advise/inform this feature is to help users who have legacy browsers that don't support unicode emojis.

Reply:
i'm okay if we don't include it, i thought it would be beneficial to provide usage context for when someone might want to use this setting

Reply:
If we're advising people to use it, then you also need to mention about the CC-BY 4.0 implications. Since we can't do the latter, we also can't advise people to use this option.